ui/00_message/common/item_info.xml

```diff
@@ -32516,63 +32553,83 @@ status debilitations.</translation>
   </item>
   <item id="13811">
     <original>細密なレリーフの刻まれたコイン</original>
+    <translation>Finely engraved coin.</translation>
```

Comment:
Hmmm yeah, but the thing is, as I understand it, engraving already implies making a relief so I thought it might be a bit redudant.

We could also go with "Coin with a finely detailed relief" since engrave is just the method to produce the relief.